### PR TITLE
Update Substrate and Fix `darwinia-fee-market`

### PR DESCRIPTION
**Pull request by AurevoirXavier**
`AccoundId` is not more defaultable.
Hence we need to change `ValueQuery` to `OptionQuery`.
Please fix the code logic. @AsceticBear

### CODE REVIEW

File: frame/fee-market/src/lib.rs [hunk 1]
```diff
@@ -138,7 +134,7 @@ pub mod pallet {
 		Blake2_128Concat,
 		T::AccountId,
 		Relayer<T::AccountId, RingBalance<T>>,
-		ValueQuery,
+		OptionQuery,
```
Comment (HackFisher) [inline frame/fee-market/src/lib.rs:137]: Do we need to migrate storage if we change to OptionQuery?

I remember that it will not affect decoding, right?

**Review by HackFisher — COMMENTED** [review]
We changed the substrate deps in this branch, but only in the scope of fee-market pallet.

Only this pallet compiles and other will fail right? Is that sufficient for the requirement of Crab/Pangolin parachain?

File: frame/fee-market/src/lib.rs (hunk 1 above)
Comment (AurevoirXavier) [reply frame/fee-market/src/lib.rs:137]: No need to.

**Comment by AurevoirXavier** [comment]
> We changed the substrate deps in this branch, but only in the scope of fee-market pallet.
> 
> Only this pallet compiles and other will fail right? Is that sufficient for the requirement of Crab/Pangolin parachain?

Yes. The parachains only need this pallet.

**Comment by AsceticBear** [comment]
> AccoundId is not more defaultable.

Can you find the upstream substrate pr related to this change? @AurevoirXavier

**Comment by AsceticBear** [comment]
Note: I upgrade the `pallet-bridge-messages` with https://github.com/darwinia-network/darwinia-bridges-substrate/commit/07bbb7427ffcf12ef9c7ce66bc2f64fb08525b08 to pass the units tests.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**